.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+---
+version: 2
+
+jobs:
+  build-tutorials:
+    docker:
+      - image: fpco/stack-build:lts-9.21
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v1-stack-{{ checksum "stack.yaml" }}
+            - v1-stack-
+
+      - run:
+          name: Upgrading stack
+          command: |
+            export PATH=$HOME/.local/bin:$PATH
```

Comment:
@juanpaucar I think we should add this line to `bashrc` just to make sure all the following commands point to the right version https://discuss.circleci.com/t/how-to-add-a-path-to-path-in-circle-2-0/11554/4

Reply:
Good observation